operator-metadata/manifests/bundle.clusterserviceversion.yaml

```diff
@@ -381,6 +381,7 @@ metadata:
     operators.operatorframework.io/internal-objects: |-
       ["strimzipodsets.core.strimzi.io"]
     olm.skipRange: '>=2.6.0-0 <2.7.0-4'
+    olm.substitutesFor: 2.7.0-4
```

Comment:
I don't understand the interplay between `olm.skipRange` and `olm.subsitutesFor` (even after reading the linked docs several times). 

A big part of what I'm missing is what does freshmaker produce? is it `2.7.0-5`? or is it something else entirely?

Reply:
We should omit `olm.subsitutesFor` and Freshmaker will add it with the version of the CSV being respun.

Reply:
> We should omit olm.subsitutesFor and Freshmaker will add it with the version of the CSV being respun.

Yes, that's my mental mode too.  See https://github.com/operator-framework/enhancements/blob/master/enhancements/substitutes-for.md to understand how Freshmaker will use it.

Reply:
I didn't understand this, but if you are confident it works this way I'll remove `olm.subsitutesFor`

Reply:
>We should omit olm.subsitutesFor and Freshmaker will add it with the version of the CSV being respun.

My reading of [substitutes-for.md](https://github.com/operator-framework/enhancements/blob/master/enhancements/substitutes-for.md) suggests multiple substitutions are allowed. So Freshmaker should cope with it being there already. The bit I'm not following is why setting `substitutesFor` is different to bumping the skipRange to the new build. 

In writing that I think I figured it out. `-<buildNumber>` effectively represents the build of "porduct code" and freshmaker is re-building the bits we "take for granted" i.e. the UBI/RHEL images. So the build number doesn't increment as "porduct code" hasn't been re-built. So the `substitutesFor ` will contain a CSV version (what ever that looks like) and not `2.7.0-4`.